ajax/libs/material-datetime-picker/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "material-datetime-picker",
+  "filename": "material-datetime-picker.min.js",
+  "description": "A Vanilla JS Material Design date/time picker component",
+  "homepage": "https://ripjar.github.io/material-datetime-picker/",
+  "keywords": [
+    "material-design",
+    "date-picker",
+    "datetime-picker",
+    "javascript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ripjar/material-datetime-picker"
+  },
+  "license": "MIT",
+  "author": "@jivings",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ripjar/material-datetime-picker.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "material-datetime-picker*"
```

Comment:
This pattern will match all the files which use `material-datetime-picker` prefix under `dist`, but if we use this filemap, we will get the `material-datetime-picker.mjs*` also.
The files `material-datetime-picker.mjs*` we don't need to host it, because it can't use in web frond-end.

@cdnjs/intern3 Would you help we check about this?
I think this should be discuss, I don't know which filemap is better, `material-datetime-picker!(.mjs)*` or `**/*picker!(.mjs)*`?
- `material-datetime-picker!(.mjs)*`: this will get the files after v2.0.0, but v1.0.1 and v1.0.0 need masually add due to different file stucture.
- `**/*picker!(.mjs)*`: this will get all the versions files.

If you have any suggestion please let me know, thanks.